public/app/features/dashboard-scene/serialization/layoutSerializers/RowsLayoutSerializer.ts

```diff
@@ -17,14 +19,26 @@ export class RowsLayoutSerializer implements LayoutManagerSerializer {
           if (layout.kind === 'RowsLayout') {
             throw new Error('Nested RowsLayout is not supported');
           }
-          return {
+          const rowKind: RowsLayoutRowKind = {
             kind: 'RowsLayoutRow',
             spec: {
               title: row.state.title,
               collapsed: row.state.isCollapsed ?? false,
               layout: layout,
             },
           };
+
+          if (row.state.$behaviors) {
+            for (const behavior of row.state.$behaviors) {
+              if (behavior instanceof RowItemRepeaterBehavior) {
+                if (rowKind.spec.repeat) {
+                  throw new Error('Multiple repeaters are not supported');
```

Comment:
In what case this could happen? 🤔

Reply:
It should never happen, the problem is there's nothing stopping it from happening. So if we create a bug somewhere that adds more than one repeater behavior we will have some weird issues where we're only persisting one of them.

Maybe this is overly defensive. One could argue that we should instead throw an error when adding a RowItemRepeaterBehavior to an array that already contains a RowItemRepeaterBehavior.